frontend/package.json

```diff
@@ -11,6 +11,7 @@
     "dev:server": "cross-env NODE_ENV=development npm run-script build:server -- --watch",
     "start": "cross-env NODE_ENV=production node ./build/express.server.js ./build/index.js",
     "format": "prettier --write ./",
+    "minify-css": "cleancss --batch --batch-suffix \".min\" ./public/theme/cdcp/css/*.css !./public/theme/cdcp/css/*.min.css",
```

Comment:
Is this really needed for a one-line css file?

Reply:
It's one line for now... who knows later?

Reply:
It's never gonna be big enough that we need to care about minification..

Reply:
- Removed clean-css.
- Created and imported `cdcp.css` stylesheet.